common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -221,6 +231,38 @@ def perform_destroy(self, instance):
         # This is not called with is_refund=True here because it is assumed the user has already been refunded.
         _process_revoke_and_unenroll_entitlement(instance)
 
+    def partial_update(self, request, *args, **kwargs):
```

Comment:
Does this set the expired_at of the entitlement to null? or to what is passed in in the update?

Reply:
Does that happen in the super.partial update at the bottom?

Reply:
Yes, it should. Haven't been able to confirm it yet though